src/python/qepsi4/_psi4.py

```diff
@@ -2,8 +2,8 @@
 
 import numpy as np
 import psi4
-from openfermion import InteractionRDM, MolecularData, general_basis_change
-from openfermion.config import EQ_TOLERANCE
+from zquantum.core.openfermion import InteractionRDM, MolecularData, general_basis_change
+from zquantum.core.openfermion.config import EQ_TOLERANCE
```

Comment:
In this case, I would actually leave the dependence on `openfermion` in this lib, as it depends not only on `QubitOperator`, but also on some other stuff which is useful, that we probably don't want to write ourselves.